thorvg/CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ endif()
 
 file(APPEND ${TVG_CROSS_CFG} "
 [host_machine]
-system = 'freertos'
+system = 'android'
```

Comment:
I recall you asked the question about this in the previous PR... Did you figure out why this has to be `android`?

Reply:
```meson
if get_option('threads') == true and host_machine.system() != 'windows' and host_machine.system() != 'android'
    thread_dep = meson.get_compiler('cpp').find_library('pthread')
    thorvg_lib_dep += [thread_dep]
endif
```

the meson build system can't find the pthread library in the default search path.  

I tried added `cpp_link_args = ['-L${CMAKE_BINARY_DIR}/esp-idf/pthread', '-lpthread']` to the mseon cross file, still no luck.